pkgs/development/interpreters/ruby/default.nix

```diff
@@ -84,8 +91,7 @@ let
           ++ (op fiddleSupport libffi)
           ++ (ops cursesSupport [ ncurses readline ])
           ++ (op zlibSupport zlib)
-          ++ (op (!atLeast31 && opensslSupport) openssl_1_1)
```

Comment:
Would this build `ruby_2_7` without openssl regardless?

Reply:
It wouldn't build `ruby_2_7`, do we still have it?

Reply:
Will fix

Reply:
Fixed.